src/DADynamoApp/DAEntrypoint.cs

```diff
@@ -0,0 +1,182 @@
+
+            Console.WriteLine("<<!>> Starting to load D4DA");
+
+            try
+            {
+                RevitServices.Transactions.TransactionManager.SetupManager(new RevitServices.Transactions.AutomaticTransactionStrategy());
+                ElementBinder.IsEnabled = true;
+
+                RevitServicesUpdater.Initialize(Updaters);
+
+                DesignAutomationBridge.DesignAutomationReadyEvent += HandleDesignAutomationReadyEvent;
+
+                Console.WriteLine("<<!>> D4DA Loaded");
+
+                return ExternalDBApplicationResult.Succeeded;
+            }
+            catch (Exception ex)
+            {
+                return ExternalDBApplicationResult.Failed;
+            }
+        }
+
+        public void HandleDesignAutomationReadyEvent(object sender, DesignAutomationReadyEventArgs e)
+        {
+            Console.WriteLine("<<!>> DA event raised.");
+            WorkItemFolder = Directory.GetCurrentDirectory();
+
+            foreach (var xx in Directory.EnumerateFiles(WorkItemFolder))
+                Console.WriteLine(xx);
+
+            var dynTempDir = Path.Combine(WorkItemFolder, "dyn_tmp");
+            Console.WriteLine($"<<!>> Work folder is '{WorkItemFolder}'");
+            var app = e.DesignAutomationData?.RevitApp;
+            Console.WriteLine("<<!>> Preparing Dynamo model. Vers 1");
+
+            var hostloc = typeof(Autodesk.Revit.ApplicationServices.Application).Assembly.Location;
+            var asmLocation = Path.GetDirectoryName(hostloc);
+            Console.WriteLine($"using asm at location {asmLocation}");
+            Console.WriteLine($"Is Loaded {LoadMessage}");
+            var curFolder = Directory.GetParent(Assembly.GetExecutingAssembly().Location).FullName;
+            var cmdArgs = new Dynamo.Applications.StartupUtils.CommandLineArguments();
+            cmdArgs.ASMPath = asmLocation;
+            cmdArgs.DisableAnalytics = true;
+            cmdArgs.ServiceMode = true;
+            cmdArgs.AnalyticsInfo = new() { HostName = "Revit_DA" };
+            cmdArgs.ImportedPaths = [Path.Combine(curFolder, "nodes", "RevitNodes.dll"), Path.Combine(curFolder, "nodes", "DSRevitNodesUI.dll")];
+            // need this for cloud, does not work on local
+            cmdArgs.UserDataFolder = Path.Combine(dynTempDir, "Dynamo Revit");
+            cmdArgs.CommonDataFolder = Path.Combine(dynTempDir, "Dynamo");
+
+            // Startup a new project, maybe an option we can have ?
+            //app.NewProjectDocument(Autodesk.Revit.DB.UnitSystem.Metric);
+
+            DocumentManager.Instance.PrepareForAutomation(app);
+
+            //true, asmLocation, "Revit_DA"
+            model = Dynamo.Applications.StartupUtils.MakeCLIModel(cmdArgs);
+
+            LoadMessage = model != null ? "loaded" : "no loaded";
+            Console.WriteLine($"Checking python in {curFolder}");
+            CheckIfPythonExists(curFolder);
```

Comment:
can you explain why you are doing this as opposed to letting Dynamo load python the normal way?